### PR TITLE
[#284] Add tests for StoriesController

### PR DESCRIPTION
**Pull request by rgenchev**
Closes https://github.com/Terrastories/terrastories/issues/284

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -82,6 +82,7 @@ group :test do
   gem 'chromedriver-helper'
   # Factories for creating database entities for testing
   gem 'factory_bot_rails'
+  gem 'rails-controller-testing'
```
Comment (namel3ss) [inline Gemfile]: Since Rails 5 it is recommended to use the built in RSpec request tests, and not use the `rails-controller-testing` gem.

**Comment by mirandawang** [comment]
@rgenchev How do you feel about the above ^ suggestions? I think it would be best to avoid introducing new dependencies if we don't need them!

**Comment by rgenchev** [comment]
> @rgenchev How do you feel about the above ^ suggestions? I think it would be best to avoid introducing new dependencies if we don't need them!

@mirandawang , I totally agree. I'll push the changes tonight. :)

File: spec/controllers/stories_controller_spec.rb [hunk 2]
```diff
@@ -1,5 +1,162 @@
...
+    end
+
+    it 'should redirect to :show' do
+      expect(subject).to redirect_to(story_path(assigns(:story)))
+    end
+  end
+
+  private
+
+  def fabricate_stories
```
[the remark this answers is not in the document]
Comment (rgenchev) [reply spec/controllers/stories_controller_spec.rb]: Done! :)

File: Gemfile (hunk 1 above)
Comment (rgenchev) [reply Gemfile]: I reverted the commit.

**Pull request merged**